src/main/java/org/cbio/gdcpipeline/model/cbio/Patient.java

```diff
@@ -13,28 +13,87 @@
 
 public class Patient extends ClinicalDataModel {
     private String patient_id;
-    private String sex;
```

Comment:
I think this was intentional. The field names match the header matches the getters and setters. If you look in the writer there is a field extractor that handles the `ClinicalDataModel` objects. I think changing this would be a little bit annoying, but I agree it's worth looking into at some point. I'll write it in as a TODO.